.env.example

```
@@ -0,0 +1,9 @@
+VITE_SUPABASE_URL=
+VITE_SUPABASE_PUBLISHABLE_KEY=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_SUPABASE_PUBLISHABLE_KEY key should go before the VITE_SUPABASE_URL key

(UnorderedKey)